### ISSUE TITLE
KRDM-Roberts Field Airport-OREGON-USA

### ISSUE THREAD

**Issue by MrMrMan** [InfiniteFlightAirportEditing/Airports#3453]
# Airport Name

Roberts Field Airport

# Country?

United States

# Improvements that need to be made?

Redone from almost scratch

# Are you working on this airport?

Yes

# Airport Priority? (A380, 10000ft+ Runway)

N/A

**Comment by MrMrMan** [comment]
Airport put on hold due to extensive construction.

**Comment by MrMrMan** [comment]
Airport still on hold due to construction. Unsure of when it will be done, airport's website has no date.

**Comment by MrMrMan** [comment]
Runway 11/23 still undergoing construction/repave/other misc construction. Still unsure of when the construction will be done. Airport will remain on hold until updated scenery is provided.

**Comment by MrMrMan** [comment]
Construction seems like it's over now! Starting work again.

**Comment by kirby2007** [comment]
I´d like to work on this one.  It would be my first airport edit.

**Comment by brunocr98** [comment]
Advisory: this airport has an unfinished runway in Google Maps, so the imagery is outdated. You may encounter some difficulties.

**Comment by kirby2007** [comment]
I think the runway completion is showing on Bing satellite imagery, so I thought that would work.  What do you think?

**Comment by Gabe-Z** [comment]
That's fine. Bing is more up to date than Google. Make sure you let Peer Reviewer know about different Imagery Satellite.

**Comment by kirby2007** [comment]
I've realized now that Bing imagery is not up-to-date either (2018?), and actually Google is more recent.  Downside with the newer Google is it shows the secondary runway in the process of being rebuilt and a few taxiways being redone as well.  Upside is that by nevertheless using the Bing imagery, the airfield will be much improved over the current version in IF (only one spawn, taxiways greatly simplified, numerous GA hangers missing, runways mislabeled).  So I will render the airfield using the Bing imagery, and then as a second phase, use Google where improvements have been completed for the terminal ramp and a few connecting taxiways.  Once this is done, it should only be a small job to update the airfield when new imagery shows the work on the secondary runway completed.

**Comment by kirby2007** [comment]
Airport completed using ESRI low-res imagery of recent changes to secondary runway and certain taxiways, however... Terminal ramp has been expanded since Google hi-res imagery, and details of this are not resolvable in the low-res, so I will continue to wait for Google imagery update before submitting.

**Comment by Czechoslovaki** [comment]
@kirby2007 Bing imagery updated to higher quality. Includes additional airline parking spots and completely paved runways.

**Comment by kirby2007** [comment]
@Czechoslovaki It certainly has!  Good catch (web version not yet updated, and I had to update to latest SAS Planet Nightly to see new imagery).  I'll add the changes and submit to PR.  Thanks!

**Comment by kilt46** [comment]
![image](https://user-images.githubusercontent.com/14023167/102721446-75a78900-42c0-11eb-8d33-d21501b89f94.png)
